packages/distribution/addon/components/inquiry-new-form.js

```diff
@@ -63,10 +63,10 @@ export default class InquiryNewFormComponent extends Component {
   *updateSearch(e) {
     e.preventDefault();
 
-    const { environment } =
-      getOwner(this).resolveRegistration("config:environment");
-
-    if (environment !== "test") {
+    /* istanbul ignore next */
```

Comment:
What does `/* istanbul ignore next */` do?

Reply:
Ignores the next block for code coverage